exonum/src/api/node/public/explorer.rs

```diff
@@ -156,8 +164,45 @@ impl ExplorerApi {
             })
     }
 
+    /// Subscribes to block commits events.
+    pub fn handle_subscribe(
+        name: &'static str,
+        backend: &mut actix::ApiBuilder,
+        shared_api_state: SharedNodeState,
+    ) {
+        let server_address = Arc::new(Mutex::new(None));
```

Comment:
1. `WsServer` initialisation looks like `lazy_static!` expansion.
Is there any reason not using https://crates.io/crates/lazy_static instead.

2. Why you create Otional server_address? Why you cant init it in this variable?

Reply:
2. Because it uses Arbiter, that need the server to be started